components/brave_wallet_ui/components/extension/confirm-transaction-panel/index.tsx

```diff
@@ -118,12 +120,21 @@ function ConfirmTransactionPanel (props: Props) {
     onConfirm,
     onReject,
     onRejectAllTransactions,
-    refreshGasEstimates,
     getERC20Allowance,
     updateUnapprovedTransactionGasFields,
     updateUnapprovedTransactionSpendAllowance,
     updateUnapprovedTransactionNonce
   } = props
+  const {
+    solFeeEstimates,
+    defaultNetworks
+  } = useSelector(({ wallet }: { wallet: WalletState }) => wallet)
+  const dispatch = useDispatch()
+
+  const transactionsNetwork = React.useMemo(() => {
```

Comment:
Extract this to its own hook. Logic is reused multiple times

Reply:
Updated!